tox.ini

```diff
@@ -0,0 +1,20 @@
+# Tox (http://codespeak.net/~hpk/tox/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+minversion = 2.2
+envlist =
+    py{27,32,33,34}-django18
+    py{27,34,35}-django110
+    py{27,34,35,36}-django111
+skip_missing_interpreters = True
+
+[testenv]
+changedir = examples/protected_downloads
```

Comment:
In #62 I've fixed the tests so they can be run at the project top level. If that lands, I don't think this `changedir` will be required.